docs-app/app/components/prose.gts

```diff
@@ -7,31 +7,34 @@ import type { TOC } from '@ember/component/template-only';
 
 const resetScroll = eModifier((element, [prose]) => {
   prose;
-  element.scrollTo(0, 0);
+  window.scrollTo(0, 0);
 });
 
-export const Prose: TOC<{ Element: HTMLDivElement }> = <template>
+export const Prose: TOC<{ Element: HTMLElement }> = <template>
   {{#let (service "docs") as |docs|}}
-    <div
-      class="grid gap-4 overflow-auto pb-8 w-fit w-full"
+    <section
+      class="flex-auto max-w-2xl min-w-0 px-4 py-16 lg:max-w-none lg:pl-8 lg:pr-0 xl:px-16"
       ...attributes
       {{resetScroll docs.selected.prose}}
     >
 
-      <div data-prose class="prose p-4" {{highlight docs.selected.prose}}>
+      <article
+        class="prose prose-slate max-w-none dark:prose-invert dark:text-slate-400 prose-headings:inline-block prose-th:table-cell prose-headings:scroll-mt-28 prose-h1:text-3xl prose-headings:font-display prose-headings:font-normal lg:prose-headings:scroll-mt-[8.5rem] prose-lead:text-slate-500 dark:prose-lead:text-slate-400 prose-a:font-semibold dark:prose-a:text-sky-400 prose-a:no-underline prose-a:shadow-[inset_0_-2px_0_0_var(--tw-prose-background,#fff),inset_0_calc(-1*(var(--tw-prose-underline-size,4px)+2px))_0_0_var(--tw-prose-underline,theme(colors.sky.300))] hover:prose-a:[--tw-prose-underline-size:6px] dark:[--tw-prose-background:theme(colors.slate.900)] dark:prose-a:shadow-[inset_0_calc(-1*var(--tw-prose-underline-size,2px))_0_0_var(--tw-prose-underline,theme(colors.sky.800))] dark:hover:prose-a:[--tw-prose-underline-size:6px] prose-pre:rounded-xl prose-pre:bg-slate-900 prose-pre:shadow-lg dark:prose-pre:bg-slate-800/60 dark:prose-pre:shadow-none dark:prose-pre:ring-1 dark:prose-pre:ring-slate-300/10 dark:prose-hr:border-slate-800 dark:prose-code:text-slate-50"
```

Comment:
I initially thought this was crazy, but upon closer inspection, this a mapping of prose labels to classes! not bad!